app/sliders/config_default.yaml

```diff
@@ -19,6 +19,17 @@ bubble:
     width: 8
     location: [0, 0]
 
+bubble_interactive:
+  figure:
+    title: Simple Bubble Plot
+    toolbar_location: null
+  linear_cmap:
+    field_name: color
+    palette: Viridis256
+  color_bar:
+    width: 8
+    location: [0, 0]
```

Comment:
Are these things ever being used somewhere? 🤔 I’m not doubting anything, but just wondering where/how these configurations are being applied.

Reply:
These specific ones are not used anymore, I will clean the config files during refactoring and get rid of them. But some others, like the ones under "figure" are passed as parameters when plotting.